packages/core/core/src/types.js

```diff
@@ -45,26 +44,22 @@ export type DependencyNode = {|
 export type FileNode = {|id: string, +type: 'file', value: File|};
 export type RootNode = {|id: string, +type: 'root', value: string | null|};
 
-export type TransformerRequestNode = {|
-  id: string,
-  +type: 'transformer_request',
-  value: TransformerRequest
-|};
 export type AssetGroupNode = {|
   id: string,
   +type: 'asset_group',
-  value: AssetGroup
+  value: AssetRequest
```

Comment:
the value for an `AssetGroupNode` is an `AssetRequest`? If so, maybe a different name or a comment explaining why?